.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -382,6 +382,45 @@ and Linux `tsh` binaries are tested for FIDO2 items.
   - [ ] `tsh touchid ls` works
   - [ ] `tsh touchid rm` works (careful, may lock you out!)
 
+### Hardware Key Login
+
+Hardware Key Login is an Enterprise feature and is not available for OSS.
+
+On Linux, hardware key support requires that `tsh` and `teleport` are both built with libpcsclite. Ask for a statically-built `tsh` and `teleport` binary for realistic tests.
+
+You will need a YubiKey 4.3+ for each test. `tsh logout` between each section.
+
+- [ ] `auth_service.authentication.require_session_mfa: hardware_key`
+  - [ ] `tsh login`
```

Comment:
I'd add here to test `tsh login` and `tsh ssh` on Windows, Linux and MacOS to test "YK discovery"